packages/account/src/Components/api-token/_tests_/api-token-clipboard.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ApiTokenClipboard from '../api-token-clipboard';
+
+const modal_root_el = document.createElement('div');
+modal_root_el.setAttribute('id', 'modal_root');
+document.body.appendChild(modal_root_el);
+
+describe('ApiTokenClipboard', () => {
+    const mock_props = {
+        scopes: ['read', 'trade', 'Admin'],
+        text_copy: 'Text Copy',
+        info_message: 'Info Message',
+        success_message: 'Success Message',
+    };
+
+    it('should render ApiTokenClipboard with the copy icon', () => {
+        render(<ApiTokenClipboard {...mock_props} />);
+        expect(screen.getByTestId('dt_copy_token_icon')).toBeInTheDocument();
+    });
+
+    it('should display "Copy this token" message when mouse enters', () => {
+        render(<ApiTokenClipboard {...mock_props} />);
+        const copy_icon = screen.getByTestId('dt_copy_token_icon');
+        userEvent.hover(copy_icon);
+        expect(screen.getByText('Info Message')).toBeInTheDocument();
```

Comment:
description says 'should display "Copy this token"' but we are checking 'Info Message' here.